science/gmtsar/Portfile

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                gmtsar
+version             5.6
+categories          science
+platforms           darwin
+license             GPL-3
+maintainers         ucsd.edu:dsandwell \
+                    ucsd.edu:xix016 \
+                    ucsd.edu:jdesanto \
+                    openmaintainer
+description         GMTSAR
+long_description    GMTSAR is a tool to process SAR data
+homepage            https://topex.ucsd.edu/gmtsar/
+master_sites        ftp://topex.ucsd.edu/pub/gmtsar/
+
+extract.suffix      .tar
+extract.cmd         tar
+extract.pre_args    -xvf
+
+distname            GMTSAR_V${version}
+distfiles           ${distname}${extract.suffix}
+
+
+checksums           rmd160  6e7a4fb9aa0ad8d35924cd79409042c6649a68ca \
+                    sha256  218ba5dfebbb47b1c227af3a13540d23c638d0bc9f8df4f13321ed70372f9f20 \
+                    size    14486016
+
+depends_lib         port:gmt5 \
+                    port:hdf5 \
+                    port:tiff
+
+default_variants    +noorbits
+
+use_autoconf        yes
+
+use_configure       yes
+
+use_parallel_build  no
+
+variant norbits description {Standard installation for S1A, ALOS, ALOS-2, CSK, TSX, and RS2 satellites} {
+}
+
+variant orbits description {Optional support for ERS and ENVISat satellites} {
+    depends_lib-append      port:gmtsar-data
+    configure.args-append   --with-orbits-dir=/opt/local/share/gmtsar-data/ORBITS
```

Comment:
please don't hardcode ```/opt/local```, use ```${prefix}``` instead; it's possible to install MacPorts in another location than /opt/local